scss/_patterns_navigation.scss

```diff
@@ -118,15 +118,9 @@
 
       & > a {
         // For when this class is applied to something that contains <a>s, e.g.:
-        // <li class="p-navigation__list"><a></a></li>
```

Comment:
This class name was either typo or some leftover, and made reading code a bit confusing.